pkg/manifests/config.go

```diff
@@ -92,6 +92,36 @@ type PrometheusK8sConfig struct {
 	VolumeClaimTemplate *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
 	RemoteWrite         []monv1.RemoteWriteSpec              `json:"remoteWrite"`
 	TelemetryMatches    []string                             `json:"-"`
+	AlertmanagerConfigs []AdditionalAlertmanagerConfig       `json:"additionalAlertManagerConfigs"`
+}
+
+type AdditionalAlertmanagerConfig struct {
+	// The URL scheme to use when talking to Alertmanagers.
+	Scheme string `json:"scheme,omitempty"`
+	// Path prefix to add in front of the push endpoint path.
+	PathPrefix string `json:"pathPrefix,omitempty"`
+	// The timeout used when sending alerts.
+	Timeout *string `json:"timeout,omitempty"`
+	// The api version of Alertmanager.
+	APIVersion string `json:"apiVersion"`
+	// TLS Config to use for alertmanager connection.
+	TLSConfig TLSConfig `json:"tlsConfig,omitempty"`
+	// Bearer token to use when authenticating to Alertmanager.
+	BearerToken string `json:"bearerToken,omitempty"`
+	// List of labeled statically configured Alertmanagers.
+	StaticConfigs []*monv1.ProbeTargetStaticConfig `json:"staticConfigs,omitempty"`
```

Comment:
I don't think we need to reuse `monv1.ProbeTargetStaticConfig` here. A `[]string` slice is enough IMHO.

Reply:
Yes. we do not have case to assign the labels to all metrics scraped from target. Will update to `[]string`